github/provider.go

```diff
@@ -4,6 +4,7 @@ import (
 	"fmt"
 	"log"
 	"time"
+	"strings"
```

Comment:
Something about this line is causing a formatting error. Can you run `gofmt` to fix it?